tests/modules/alert.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import { render, screen } from "@testing-library/react";
+import { Alert, AlertProps } from "../../src";
+import { BaseColorSet } from "../../src/spec/color-spec/base-color-set";
+import { DEFAULT_TEXT } from "../common";
+
+const renderComponent = (props?: Partial<AlertProps>) => {
+    return render(
+        <Alert type="success" {...props}>
+            {props?.children || DEFAULT_TEXT}
+        </Alert>
+    );
+};
+
+const getActionLink = (isQuery = false) => {
+    if (isQuery) {
+        return screen.queryByRole("link");
+    }
+    return screen.getByRole("link");
+};
+
+describe("Alert", () => {
+    beforeEach(() => {
+        jest.resetAllMocks();
+    });
+
+    it("should render the component", () => {
+        renderComponent();
+
+        expect(screen.getByText(DEFAULT_TEXT)).toBeInTheDocument();
+    });
+
+    describe("type", () => {
+        it.each`
+            type         | backgroundColor                              | borderColor
+            ${"success"} | ${BaseColorSet.Validation.Green.Background}  | ${BaseColorSet.Validation.Green.Border}
+            ${"warning"} | ${BaseColorSet.Validation.Orange.Background} | ${BaseColorSet.Validation.Orange.Background}
+            ${"error"}   | ${BaseColorSet.Validation.Red.Background}    | ${BaseColorSet.Validation.Red.Background}
+        `(
+            "should render background $backgroundColor with border $borderColor for $type type",
+            ({ type, backgroundColor, borderColor }) => {
+                renderComponent({ type });
+
+                expect(screen.getByText(DEFAULT_TEXT)).toHaveStyle({
+                    backgroundColor,
+                    borderColor: `2pt solid ${borderColor}`,
+                });
+            }
+        );
+    });
+
+    describe("actionLink", () => {
+        it("should render action link if the prop is provided", () => {
+            renderComponent({ actionLink: { href: "www.google.com" } });
+
+            expect(getActionLink()).toBeInTheDocument();
```

Comment:
would be good to check the link name to verify that the `actionLink.children` is rendered too